ubuntu/latest.Dockerfile

```
@@ -7,13 +7,151 @@
+### SDL2_image package
+### Boost C++ Libraries
+### Install from source
+# ========================================================================================================
+# curl -LO https://github.com/acaudwell/Gource/releases/download/gource-0.55/gource-0.55.tar.gz
+# tar -xzf gource-0.55.tar.gz
+# cd gource-0.55
+# ./configure
+# make
+# sudo make install
+# ========================================================================================================
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)